scripts/build-manylinux-wheels.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -e -x
+
+# This is to be run by docker inside a docker image.
+# You can test it locally on a Linux machine by installing docker and running from this repo's root:
+# $ sudo chmod 777 scripts/build-manylinux-wheels.sh
+# $ sudo docker run  -e PLAT=manylinux1_x86_64  -v `pwd`:/io  quay.io/pypa/manylinux1_x86_64   /io/scripts/build-manylinux-wheels.sh
```

Comment:
Is `sudo` really needed for these?

Reply:
It is on my machine. I can't get Docker to run even the most harmless of commands without sudo. I take it that it shouldn't be necessary?

Reply:
It'll depend on the individual user's machine, so let's leave it out. It's just a comment, and people can add `sudo` if they need to. Anyway, the main use case is running automatically from the CI.